packages/fela-utils/src/generateCSSSelector.js

```diff
@@ -2,7 +2,10 @@
 export default function generateCSSSelector(
   className: string,
   pseudo: string = '',
-  specificityPrefix?: string = ''
+  specificityPrefix?: string = '',
+  propertyPriority?: number = 1
 ): string {
-  return `${specificityPrefix}.${className}${pseudo}`
+  const classNameSelector = `.${className}`.repeat(propertyPriority)
```

Comment:
I don’t know how safe this is is going to be if I’m being honest. I was hoping we could solve that at the resolver level rather than within the stylesheet.

Reply:
Safe in what regard?